_activities/forcesfriction.md

```diff
@@ -1,30 +1,16 @@
 ---
 layout: activity
+lang: en
 #
 # Content
 #
 title: "Forces and Friction"
-title_fr: "Forces et Friction"
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)